server/src/main/java/com/thoughtworks/go/server/MaterialTestConnectionToggle.java

```diff
@@ -0,0 +1,27 @@
+/*
+ * Copyright 2020 ThoughtWorks, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package com.thoughtworks.go.server;
+
+import com.thoughtworks.go.server.service.support.toggle.Toggles;
+
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+
+public class MaterialTestConnectionToggle extends SparkOrRailsToggle {
```

Comment:
we don't need a separate class for this, we can add a method inside `SparkOrRailsToggle` class